release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,10 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 16 October 2024 - Enhancement - Admin app - Usage Export (CSV) [ID 41117]
+
+In the [Admin app](https://admin.dataminer.services), a new feature has been introduced on the *Usage* page, allowing users to export usage metrics in a CSV file. Clicking the *Export usage* button will initiate the export process. A pop-up window will appear, where you can choose the separator to be used in the CSV file, as well as whether to include column titles at the top of the exported CSV file. Once the file has been generated, you will receive an email containing a link to download the CSV file. The download link included in the email will be valid for a period of 7 days.
```

Comment:
Some suggestions

```suggestion
In the [Admin app](https://admin.dataminer.services), a new feature was introduced on the *Usage* page, allowing users to export usage metrics to a CSV file. Clicking the *Export usage* button will initiate the export process. A pop-up window will appear, where you can choose the separator for the CSV file, as well as whether to include column titles at the top of the exported CSV file. Once the file has been generated, you will receive an email containing a link to download the CSV file. The download link included in the email will be valid for a period of 7 days.
```

Reply:
"has been" is better than "was" here: present perfect should be used when you're talking about a time period that begins at some point in the past and extends up to now (rather than a specific point in the past), especially when the focus is on the present result.
More on this here: [https://www.scribbr.com/verbs/present-perfect/](https://www.scribbr.com/verbs/present-perfect/).